docs/pages/get-started.mdx

```diff
@@ -212,81 +209,45 @@ $ tsh login --proxy=<Var name="teleport.example.com" /> --user=teleport-admin
   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
 ```
 
-## Step 5/6. Access your server
-
-Since You configured the Teleport SSH Service, you can easily access your Linux
-host after logging in to Teleport.
-
-See the logins you can use to access your server:
-
-```code
-$ tsh status
-> Profile URL:        https://teleport.example.com:443
-  Logged in as:       teleport-admin
-  Cluster:            teleport.example.com
-  Roles:              access, editor
-  Logins:             root, ubuntu, ec2-user
-  Kubernetes:         enabled
-  Valid until:        2022-04-26 04:55:59 -0400 EDT [valid for 11h38m0s]
-  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
-```
-
-List all SSH servers connected to Teleport:
-
-```code
-$ tsh ls
-Node Name        Address        Labels
----------------- -------------- -------------------------------------
-mynode 127.0.0.1:3022 env=example,hostname=mynode
-```
-
-SSH into `mynode` as `root`:
 
-```code
-$ tsh ssh root@mynode
-```
-
-Once you connect to your server, execute some commands in your shell so you can
-view a recording of your session later.
+</Details>
 
-## Step 6/6. Play back your session
+## Step 4/4. Enroll your infrastructure
 
-Servers you have registered with Teleport record the commands that users execute
-during their sessions so operators can play them back later to investigate
-issues.
+With Teleport, you can protect all of the resources in your infrastructure
+behind a single identity-aware access proxy, including servers, databases,
+applications, Kubernetes clusters, Windows desktops, and cloud provider APIs.
 
-In the Teleport Web UI, click the dropdown menu on the upper left of the screen,
-then choose **Management**. click the **Activity** tab in the sidebar on the
-left of the screen, then click **Session Recordings**. You will see your session
-from the previous step:
+To enroll a resource with Teleport, visit the Web UI and click the name of a
+resource on the sidebar. The Web UI will show you the steps you can take to
+enroll that resource.
```

Comment:
Can we make this more explicit by referencing the **Add `$RESOURCE`** button(s)?